lib/plugins/ray_trace.js

```diff
@@ -23,4 +24,39 @@ module.exports = (bot) => {
 
     return bot.world.raycast(eyePosition, viewDirection, maxDistance, matcher)
   }
+
+  bot.entityAtCursor = (maxDistance = 3.5) => {
+    const block = bot.blockAtCursor(maxDistance)
+    maxDistance = block?.intersect.distanceTo(bot.entity.position) ?? maxDistance
+
+    const entities = Object.values(bot.entities)
+      .filter(entity => entity.type !== 'object' && entity.username !== bot.username && entity.position.distanceTo(bot.entity.position) <= maxDistance)
+
+    const dir = new Vec3(-Math.sin(bot.entity.yaw) * Math.cos(bot.entity.pitch), Math.sin(bot.entity.pitch), -Math.cos(bot.entity.yaw) * Math.cos(bot.entity.pitch))
+    const iterator = new RaycastIterator(bot.entity.position.offset(0, bot.entity.height, 0), dir.normalize(), maxDistance)
+
+    let targetEntity = null
+    let targetDist = maxDistance
+
+    for (let i = 0; i < entities.length; i++) {
+      const entity = entities[i]
+      const w = entity.width / 2
+
+      const shapes = [[-w, 0, -w, w, entity.height + (entity.type === 'player' ? 0.18 : 0), w]]
```

Comment:
where does the 0.18 constant come from?

Reply:
Adding the 0.18 constant comes is due to [this](https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/physics.js#L248) where the bot's height is set to its eye height, 1.62, rather than the bot's actual height 1.80.